doc/install/mne_tools_suite.rst

```diff
@@ -30,45 +30,22 @@ Related software
   C++ and is primarily intended for embedded and real-time applications.
 
 There is also a growing ecosystem of other Python packages that work alongside
-MNE-Python, including packages for:
+MNE-Python, including:
+
+.. dropdown:: Related Python ecosystem packages
+   :open:
+   :icon: database
+   :chevron: down-up
```

Comment:
my feeling is don't put this in a dropdown. It's slick, but it messes up the related sidebar positioning (now the sidebar comes *after* this section instead of beside it)